### PR TITLE
Added classpath as an argument to jarcat

### PR DESCRIPTION
**Pull request by BNMetrics**
As the manifest file to compile scala to jar files will need a classpath referencing to the scala std library

### CODE REVIEW

File: tools/jarcat/main.go [hunk 1]
```diff
@@ -161,6 +162,8 @@ func main() {
 	f.Align = opts.Zip.Align
 	f.Prefix = opts.Zip.Prefix
 
+	must(f.AddManifest(opts.Zip.MainClass, opts.Zip.ClassPath))
```
Comment (peterebden) [inline tools/jarcat/main.go]: this should still be conditional - before it only wrote it if main_class was passed, now it should only write if main_class or class_path are passed.

File: tools/jarcat/zip/writer.go [hunk 2]
```diff
@@ -338,8 +338,20 @@ func (f *File) AddInitPyFiles() error {
 }
 
 // AddManifest adds a manifest to the given zip writer with a Main-Class entry (and a couple of others)
-func (f *File) AddManifest(mainClass string) error {
-	manifest := fmt.Sprintf("Manifest-Version: 1.0\nMain-Class: %s\n", mainClass)
+func (f *File) AddManifest(mainClass string, classPath string) error {
+
+	if mainClass == "" && classPath == "" {
```
Comment (peterebden) [inline tools/jarcat/zip/writer.go]: not sure about this. I think this function should always write the manifest, the decision of whether to do it or not should be done by the caller.

**Comment by peterebden** [comment]
Is this still a thing we need? The Scala stuff is working fine without, right?

**Comment by BNMetrics** [comment]
Yeah, we don't actually need this anymore

**Pull request closed without merging**